### PR TITLE
Group drawn and data group markers together in the same clusters

### LINKED ISSUE #153: Drawn, map and layer icons do not group resulting in strange UX
## Description
Currently grouping of marker icons is effective when markers are all drawn or all in data layers. However when they are a mix of the two types they do not group. This results in strange behaviour making it hard to select icons.

Would it be possible to expand clustering so that it functions between sets?

https://user-images.githubusercontent.com/8669848/205033774-6dd232e0-1138-4dd0-a0d5-0003f74a0b65.mp4

## Acceptance Criteria

- Markers cluster together regardless if they are drawn, saved to a map or saved to a data group.

### PR DESCRIPTION
**Pull request by King-Mob**
#### What? Why?

Closes #153 

Moved the data group markers generation to the drawn markers component

#### What should we test?

Turn on a data group with markers in, draw a marker next to a data group marker, watch the cluster form.

#### Deployment notes

None

### CODE REVIEW

File: src/components/MapboxMap.js [hunk 1]
```diff
@@ -314,7 +314,15 @@ class MapboxMap extends Component {
             <MapProperties center={lngLat} map={this.map} />
           )}
           {/* Markers */}
```
Comment (rogup) [inline src/components/MapboxMap.js:316]: I think it might be worth saying in the comment that this includes data group markers

Comment (rogup) [reply src/components/MapboxMap.js:316]: And saying in the comment for the above MapDataGroups component that it excludes markers, due to the clustering issue

**Review by rogup — CHANGES_REQUESTED** [review]

File: src/components/MapboxMap.js (hunk 1 above)
Comment (King-Mob) [reply src/components/MapboxMap.js]: Done

**Comment by lin-d-hop** [comment]
LGTM. Great work :raised_hands:

**Pull request merged**